addons/website_sale/controllers/main.py

```diff
@@ -1227,3 +1227,50 @@ def get_combination_info_website(self, product_template_id, product_id, combinat
                 'product_variant': request.env['product.product'].browse(res['product_id']),
             }))
         return res
+
+    # ------------------------------------------------------
+    # Products Search Bar
+    # ------------------------------------------------------
+
+    def get_product_searchbar_domain(self, term, product_options, **kwargs):  # set to use old function with cat and attr
+        domain = request.website.sale_product_domain()
```

Comment:
Add docstring to your functions ;)
Kinda like I did here: https://github.com/odoo/odoo/pull/29624/commits/f6963e69e16f7eafa3c3b1fca336c933808fd90a#diff-1c474d1d26f2af9011f6533a9b97df61R115